frontend/helpers/path-helpers.ts

```diff
@@ -49,6 +62,76 @@ export function productListPath(
    }
 }
 
+interface AkineoProductUrlProps {
```

Comment:
I think all references to `akineo` should be `akeneo`

Reply:
Massimo won't be able to make this change today, but if you want to make this change yourself go for it.